electron/src/js/squirrel.ts

```diff
@@ -38,7 +40,13 @@ const exeName = `${config.NAME}.exe`;
 const linkName = `${config.NAME}.lnk`;
 
 const taskbarLink = path.resolve(
-  path.join(process.env.APPDATA, 'Microsoft', 'Internet Explorer', 'Quick Launch', 'User Pinned', 'TaskBar', linkName)
+  process.env.APPDATA || '',
+  'Microsoft',
+  'Internet Explorer',
+  'Quick Launch',
+  'User Pinned',
+  'TaskBar',
+  linkName
 );
```

Comment:
This is getting out of hand. Let's move app data into a const. Then create the path. And finally do another path that adds the link name to it.